test/test_jit.py

```diff
@@ -9391,7 +9389,12 @@ def forward(self, x, y):
     'test_nn_cross_entropy',
     'test_nn_interpolate',
     'test_nn_fold',
+}
+
+EXCLUDE_PYTHON_PRINT = {
```

Comment:
what is this list actually for? did not see where we have python print for those functionals.

Reply:
It happens for every JitTestCase, but _unpool_output_size was having some jitter problems